test/equivalences.jl

```diff
@@ -11,7 +11,7 @@
     @testset "exact posterior" begin
         # There is a closed form optimal solution for the variational posterior
         # q(u) (e.g. # https://krasserm.github.io/2020/12/12/gaussian-processes-sparse/
-        # equations (11) & (12)). The SVGP posterior with this optimal q(u)
+        # equations (11) & (12)). The SparseVariationalApproximation posterior with this optimal q(u)
```

Comment:
This is one case where I would leave SVGP, as I think this is referring to the model (paper), not to the struct.
```suggestion
        # equations (11) & (12)). The SVGP posterior with this optimal q(u)
```